tasks/configure_user.yaml

```diff
@@ -0,0 +1,41 @@
+---
+- name: "check for required fact - name"
+  fail:
+    msg: "missing required fact: name"
+  with_items: "{{ users }}"
+  when: ( user.name is not defined )
+  loop_control:
+    loop_var: user
+  delegate_to: localhost
+
+- name: "display input"
+  debug:
+    var: users
+  delegate_to: localhost
+
+- name: "fetch existing user account details"
+  cli:
+    command: show running-config | section user
```

Comment:
I am not sure if section is valid in older iOS. any reason we can not use 'include' ?

Reply:
Yes, actually a part of config lines including the key-hash does not come up when using `include` instead of section. How should we tackle this then?